### ISSUE TITLE
Error while generating list of VMWare adapters in use.

### ISSUE THREAD

**Issue by sp1icer** [hashicorp/vagrant#11815]
### Vagrant version

Vagrant 2.2.9

### Vagrant Utility version

1.0.10

### VMWare Workstation version

VMWare Workstation Pro 15.5.6 build-16341506

### Host operating system

Windows 10

### Guest operating system

Any; in this case, bento/ubuntu-20.04

### Vagrantfile

```ruby
Vagrant.configure("2") do |config|
  config.vm.box = "bento/ubuntu-20.04"
  config.vm.provider "vmware_desktop" do |vb|
    vb.gui = true
    # Customize the amount of memory on the VM:
    vb.memory = "1024"
  end
end
```

### Debug output

https://gist.github.com/sp1icer/5a7dd4a2200694c2717fdc66ad830644

### Expected behavior

A Ubuntu VM is started.

### Actual behavior

I receive the following output:
```
An error was encountered while generating the current list of
available VMware adapaters in use on this system.

  Get http://localhost:1543/api/vmnet: GET http://localhost:1543/api/vmnet giving up after 5 attempts

Please resolve any problems reported in the error message above and
try again.
```

### Steps to reproduce

1. Install VMWare, Vagrant, vagrant-vmware-utility
1. vagrant init bento/ubuntu-20.04
2. change provider from "virtualbox" to "vmware_desktop" in Vagrantfile
3. vagrant up

### References

None that I saw listed in open/closed - may have missed as I was trying to fix quickly.

### Extra notes

I've tested against VMWare and VirtualBox providers with the same configuration - VB successfully starts, whereas VMWare fails with the error output above. I've also re-installed VMWare, Vagrant, and the Vagrant-VMWare plugin. I've re-activated the plugin, as well as deleting .vmware.d from my $HOME. I've even removed and re-created the VMWare adapters via the Virtual Network Editor. Nmap doesn't show the VMWare API running on 1543, though I can manually start it with vmrest.exe but it appears on a different port. Unsure on this, but it seems the port changes after restarting the vagrant-vmware-utility.

Note: Re-installing an older version is confirmed to work with everything else remaining the same as before (utility ver. 1.0.9).

**Comment by chrisroberts** [comment]
@sp1icer Thank you for opening this issue. I was able to reproduce the behavior and have implemented a fix for the underlying issue. This fix is included in the latest release of the utility (1.0.11) which is available for download now: https://www.vagrantup.com/vmware/downloads.html

Thank you again for reporting this issue and please let me know if you run into any other issues.

Cheers!

**Comment by sp1icer** [comment]
@chrisroberts - I'll get the updated utility later today and I'll test it out. I'll reach back out if I find anything else.

Thanks so much for the quick fix!
